src/03-control-flow.js

```diff
@@ -28,7 +35,13 @@ function isEqual(a, b) {}
  *  findLarger(19, 7);
  *  //> 19
  */
-function findLarger(a, b) {}
+function findLarger(a, b) {
```

Comment:
remember that here we know that if `a > b` we will return true.  Because of this we actually don't need an `else` block at all as the only time we will reach the else block is if `b >= a`.